deploy/heroku-deploy/Dockerfile

```diff
@@ -13,6 +13,20 @@ RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-${
 # see installation instructions: https://devcenter.heroku.com/articles/heroku-cli#standalone-installation
 RUN curl https://cli-assets.heroku.com/install-standalone.sh | sh
 
+# Install AWS CLI
```

Comment:
Why are we doing this in the `heroku-deploy` Dockerfile when this is for deploying outside Heroku? Are we cornered into it by the build process? Would it make sense to split this into a separate Dockerfile for deploying to AWS?

Reply:
We are mostly cornered into how `panorama_command` is used in the deployment process. In NDS's `pipeline.yml` file for Buildkite steps, we call `panorama_command` to build and run a separate Docker container that runs `heroku` commands. I think renaming the file might make more sense, but this PR's change simply installs the AWS CLI instead of deploying to AWS ECR. The deployment to ECR will happen in [INFRA-2403](https://panoramaed.atlassian.net/browse/INFRA-2403)

Reply:
The other reason for installing the cli here is that our Heroku credentials used in the image from this Dockerfile are stored in an `env` file inside an encrypted S3 bucket. We will update this `env` file with AWS CLI credentials.